topics/climate/tutorials/ocean-variables/tutorial.md

```diff
@@ -223,51 +228,114 @@ Now, if you have finished with your analysis you can exit ODV. To do so you need
+Use ODV outputs as DIVAnd input.
 > <hands-on-title>Run DIVANnd</hands-on-title>
 >
 > 1. Use {% tool [DIVAnd](interactive_tool_divand) %} with the following parameters:
 >    - *"Do you already have a notebook"*: `Start with a fresh notebook`
->    - *"Include data into the environment"*: `data_from_Eutrophication_Med_profiles_2022_unrestricted`
+>    - *"Include data into the environment"*: `ODV netcdf outputs` (output of **Ocean Data View** {% icon tool %})
 > 2. **Run tool**
 > 3. {% snippet faqs/galaxy/interactive_tools_open.md tool="DIVAnd" %}
 {: .hands_on}
 
-Now that you are in your jupyterlab with the right environment to use DIVAnd and a set of notebooks (in the folder **notebooks**) to guide you, you can start the rest of your analysis.
-You can find your data from ODV in the **data** folder of the jupyterlab. 
+> <hands-on-title>Navigate the jupyterlab</hands-on-title>
+> 1. Once in the Jupyterlab, go to the **Notebooks** folder.
+> 2. There open a terminal window and write :
+>    ```bash
+>    wget https://raw.githubusercontent.com/fair-ease/Divand-Galaxy-ext/main/90-analysis-shorten.ipynb
+>    ```
+> 3. Run the entire notebook.
+>
+> In the first 'outputs' folder (at the root of the Jupyterlab folders), you should now have one netcdf file "Water_body_Phosphate_Mediterranean".
+>
+> So, you can now properly close the Jupyter notebook.
+>
+> 4. On the top left click on **files** and then on **Shut down**.
+> 5. Go back to your Galaxy instance.
+{: .hands-on}
 
-Once you are done you have to save all your wanted data and visualisation in the **outputs** folder and then go to the top left in the **file** section and click on **Exit**.
+After a couple of minutes, your outputs should appear in your Galaxy history.
 
-After a couple of minutes, your outputs should appear in your Galaxy history. 
+# BONUS: Even more ODV
 
+Do you want to go even further and visualise your new netcdf data? (Because this tutorial is fascinating and you want to keep reading ;))
+
+> <hands-on-title>Open your Netcdf data in ODV</hands-on-title>
+>  {% tool [Ocean Data View](interactive_tool_odv) %} with the following parameters:
+>    - *"Select if you are using an ODV collection in a zip folder or if you have your own raw data"*: `The data you are using are Netcdf or text tabular files`
+>    - *"Netcdf or tabular text file. For text file, odv format is recommanded."*: `DIVAnd netcdf outputs` (output of **DIVAnd** {% icon tool %})
+>
+> Click on **Run Tool**
+>
+> {% snippet faqs/galaxy/interactive_tools_open.md tool="ODV" %}
+>
+> 1. Click on close of the pop-up screen for the check for Updates
+> 2. Go the top left and click on **File**, then on **Open...**
+> 3. On the pop-up screen on the left panel select **ODV**, then the folder **galaxy**, then **data**. 
+> You should see a file open it (double clicking)
+> 4. Click on **Open** in the bottom right. Then click on **Next** every time and **Finish** you can keep everything by default.
```

Comment:
@Marie59 I did not understand this sentence. What should I do with "Finish"?

Reply:
Click on it ;)